test/ctia/entity/event_test.clj

```diff
@@ -185,18 +177,16 @@
                 (testing "event timeline should contain all actions by user, with respect to their visibility"
 
                   (is (= '(1 3) (map :count timeline1-body)))
-                  (is (every? #(= "user1" (:owner %))
-                              timeline1-body))
-                  (is (every? #(= "user1" (:owner %))
-                              timeline2-body))
+                  (is (= (->> timeline1-body (map :owner) count)
+                         (->> timeline1-body (map :owner) set count))
+                      "owners should differ")
```

Comment:
I would be explicit on the expected list to also check that TLP is repected (as far as I checked only events from `group1` should be in this list).
```suggestion
                  (is (= (= #{"user1" "user2}
                               (set (map :owner timeline1-body))) 
                           "owners should differ")
```